storey/dataframe.py

```diff
@@ -26,8 +26,6 @@ class ReduceToDataFrame(Flow):
     :param columns: List of column names to be passed as-is to the DataFrame constructor. Optional.
     :param insert_key_column_as: Name of the column to be inserted for event keys. Optional.
         If not set, event keys will not be inserted into the DataFrame.
-    :param insert_time_column_as: Name of the column to be inserted for event times. Optional.
```

Comment:
I think should still be there - just need to change to insert_processing_time_column_as